lib/configcat/rolloutevaluator.rb

```diff
@@ -15,25 +15,28 @@ def initialize(log)
     def evaluate(key:, user:, default_value:, default_variation_id:, settings:)
       setting_descriptor = settings[key]
       if setting_descriptor === nil
-        error = "Evaluating get_value('%s') failed. Value not found for key '%s'. Returning default_value: [%s]. Here are the available keys: %s" % [key, key, default_value.to_s, settings.keys.join(", ")]
-        @log.error(error)
+        error = "Failed to evaluate setting '#{key}' (the key was not found in config JSON). Returning the `default_value` parameter that you specified in your application: '#{default_value}'. Available keys: [#{settings.keys.join(", ")}]."
```

Comment:
Available keys should be surrounded with single quotes like `'Key1', 'Key2', 'Key3'`